Makefile

```diff
@@ -89,3 +90,12 @@ antora-preview: antora
 	@echo Preveiew should be available at http://localhost:5000
 	@echo ------------
 	podman run --rm -it -v $(CURDIR)/public:/usr/share/nginx/html:ro -p 5000:80 nginx
+
+build_container: Dockerfile
+	$(DOCKER) build . -t howto
+
+OUTPUT/modules: Makefile
+	$(DOCKER) run -v "$$(pwd)/$@:/BUILD/modules" -it howto sh -c 'cd BUILD && make antora'
+
+OUTPUT/index.html: Makefile
+	$(DOCKER) run -v "$$(pwd)/OUTPUT/:/BUILD/OUTPUT" -it howto sh -c 'cd BUILD && make && cp index.html OUTPUT'
```

Comment:
Why are you coping index.html to OUTPUT? Can't you mount $(CURDIR) as /BUILD and keep all generated files in /BUILD?